command/kv_test.go

```diff
@@ -262,7 +262,7 @@ func TestKVPutCommand(t *testing.T) {
 		}
 		combined := ui.OutputWriter.String() + ui.ErrorWriter.String()
 		if !strings.Contains(combined, "Success!") {
-			t.Errorf("expected %q to contain %q", combined, "created_time")
+			t.Errorf("expected %q to contain %q", combined, "Success!")
```

Comment:
This looks like a typo I caught where we check the output for the words "Success!" but then if not found, the outputted error said "created_time" instead.